src/get_env.c

```diff
@@ -21,10 +21,11 @@ extern "C"
 #include <stdlib.h>
 
 #include "rcutils/get_env.h"
+#include "rcutils/thread.h"
 
 #ifdef _WIN32
 # define WINDOWS_ENV_BUFFER_SIZE 2048
-static char __env_buffer[WINDOWS_ENV_BUFFER_SIZE];
+THREAD_LOCAL_STORAGE static char __env_buffer[WINDOWS_ENV_BUFFER_SIZE];
```

Comment:
This may cause memory allocations where before they did not occur, see:

https://github.com/ros2/rcutils/blob/e70143a537a9e1540f77cdb74848346cd11f6c59/include/rcutils/error_handling.h#L101-L141